styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
No, but something like that, I'd rather use an SVG so you can scale it and keep it clean.  But we could add support for it I guess if we need to.